cmd/sshpiperd/main.go

```diff
@@ -157,6 +163,10 @@ func main() {
 
 			log.SetLevel(level)
 
+			if ctx.String("log-format") == "json" {
+				log.SetFormatter(&log.JSONFormatter{})
```

Comment:
what about return an error for invalid values?

Reply:
I've added some validation. For some reason I was convinced urfave was going to handle that automatically.. 😅 Let me know if that looks okay.